itest/assertions.go

```diff
@@ -1913,7 +1913,9 @@ func AssertAssetBalances(t *testing.T, client taprpc.TaprootAssetsClient,
 
 	require.Equal(t, len(allAssets), len(assetIDBalances.AssetBalances))
 
+	var totalBalance uint64
```

Comment:
nit: shouldn't this commit precede the previous `itest: test balances with an without leased flag`